src/Libraries/CoreNodeModels/Remember.cs

```diff
@@ -0,0 +1,126 @@
+using Dynamo.Graph;
+using Dynamo.Graph.Nodes;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using VMDataBridge;
+
+namespace CoreNodeModels
+{
+    [NodeName("Remember")]
+    [NodeDescription(nameof(Properties.Resources.RememberDescription), typeof(Properties.Resources))]
+    [NodeCategory("Core.Data")]
+    [InPortNames(">")]
+    [InPortTypes("object")]
+    [InPortDescriptions(typeof(Properties.Resources), nameof(Properties.Resources.RememberInputToolTip))]
+    [OutPortNames(">")]
+    [OutPortTypes("object")]
+    [OutPortDescriptions(typeof(Properties.Resources), nameof(Properties.Resources.RememberOuputToolTip))]
+    [IsDesignScriptCompatible]
+    [DynamoServices.RegisterForTrace]
+    public class Remember : NodeModel
+    {
+        private string cache = "";
+
+        [JsonProperty("Cache")]
```

Comment:
why do you need this when the property has the same name?

Reply:
Good catch